### PR TITLE
Combined PR of KaleoFormsNewProcess testcases: LRQA-70678, LRQA-70680, LRQA-70681, LRQA-70682, LRQA-70683, LRQA-70684, LRQA-70685, LRQA-70686

### PR DESCRIPTION
**Pull request by alessandromalves**
https://issues.liferay.com/browse/LRQA-70644

### CODE REVIEW

**Comment by alessandromalves** [comment]
ci:test:sf

**Review by lucasperj — APPROVED** [review]

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsNewProcess.testcase [hunk 1]
```diff
@@ -41,21 +41,77 @@ definition {
...
+			kfProcessName = "Kaleo Forms Process",
+			workflowTask = "created");
+
+		KaleoFormsAdmin.addForm(kfFormName = "Created Task Form");
+
+		KaleoFormsAdmin.saveForm();
+
+		KaleoFormsAdmin.chooseForm(kfFormName = "Created Task Form");
+
+		KaleoFormsAdmin.saveProcess(kfProcessName = "Kaleo Forms Process");
```
Comment (lucasperj) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsNewProcess.testcase:76]: You can create a macro to add a workflow. Thank you!

**Review by lucasperj — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/macros/KaleoFormsAdmin.macro [hunk 2]
```diff
@@ -608,6 +608,77 @@ definition {
...
+
+			KaleoFormsAdmin.chooseForm(kfFormName = "${kfFormName}");
+
+			AssertElementPresent(locator1 = "Button#SAVE");
+
+			KaleoFormsAdmin.saveProcess(kfProcessName = "${kfProcessName}");
+
+			Alert.viewSuccessMessage();
+		}
+	}
```
Comment (lucasperj) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/macros/KaleoFormsAdmin.macro:680]: Very good, this is a very interesting macro Alessandro, congratulations!

**Comment by lucasperj** [comment]
ci:test:sf

**Review by lucasperj — APPROVED** [review]

**Comment by lucasperj** [comment]
ci:forward

**Pull request closed without merging**